App.js

```diff
@@ -5,8 +5,30 @@ import * as Font from 'expo-font';
 import React, { useState } from 'react';
 import { Platform, StatusBar, StyleSheet, View } from 'react-native';
 import { SafeAreaProvider } from 'react-native-safe-area-context';
+import * as Sentry from 'sentry-expo';
 import AppNavigator from './navigation/AppNavigator';
 
+Sentry.init({
+  dsn: 'https://dacd32167a384e189eab16e9588c0e67@sentry.io/5172575',
+  enableInExpoDevelopment: true,
+  debug: true,
```

Comment:
For production do I just turn both of these to false?

Reply:
I think `enableInExpoDevelopment` is specifically for Expo to be able to interact with Sentry.. probably not necessary in production. `debug` can be turned off in production.